apps/main/src/dao/components/PageAnalytics/VeCrvFeesChart/FeesBarChartTooltip.tsx

```diff
@@ -4,40 +4,31 @@ import { TooltipProps } from 'recharts'
 import styled from 'styled-components'
 import { t } from '@lingui/macro'
 
-import { formatNumber, convertToLocaleTimestamp } from '@ui/utils/utilsFormat'
+import { convertToLocaleTimestamp, formatDateFromTimestamp, formatNumber } from '@ui/utils/utilsFormat'
 
 import Box from '@ui/Box'
+import type { Distribution } from '@curvefi/prices-api/revenue'
 
 const FeesBarChartTooltip: React.FC<TooltipProps<ValueType, NameType>> = ({ active, payload }) => {
-  const currentTime = convertToLocaleTimestamp(new Date().getTime() / 1000)
-
   if (active && payload && payload.length) {
-    const { date, fees_usd, timestamp } = payload[0].payload
-    const payloadTimestamp = convertToLocaleTimestamp(new Date(timestamp).getTime() / 1000)
+    const { feesUsd, timestamp } = payload[0].payload as Distribution
 
     return (
       <TooltipWrapper>
         <Box flex flexColumn flexGap={'var(--spacing-1)'}>
           <TooltipColumn>
             <TooltipDataTitle>{t`Distribution Date`}</TooltipDataTitle>
-            {date ? (
-              <TooltipData>
-                {date}
-                {payloadTimestamp > currentTime && <strong> {t`(in progress)`}</strong>}
-              </TooltipData>
-            ) : (
-              <TooltipDataNotAvailable>{t`N/A`}</TooltipDataNotAvailable>
-            )}
+            <TooltipData>
+              {formatDateFromTimestamp(convertToLocaleTimestamp(timestamp.getUTCTimestamp()))}
```

Comment:
This looks cumbersome. Why do we need unix timestamps? The JS date object already supports the locale format and `timestamp` is a date object
```suggestion
              {formatDate(timestamp)}
```